src/component/navigation/Navigation.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import  {Link}  from "react-router-dom";
+
+import { style } from "./style";
```

Comment:
make it consistent through out the project
styles.js somewhere, somewhere style.js

use one